examples/events/EiffelServiceStartedEvent/simple.json

```diff
@@ -0,0 +1,17 @@
+{
+  "meta": {
+    "type": "EiffelServiceStartedEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee5",
+    "source": {
+    "domainId": "example.domain"
+    }
+  },
+    "links": [
+      {
+      "type": "ENVIRONMENT",
+      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
+      }
```

Comment:
This needs to reference an artifact deployment, too, wouldn't you agree? Yes, you can infer what the service is by looking at what has been deployed to the environment, but it's better to be explicit. Besides, consider the scenario where a single host (the ENVIRONMENT) holds a number of services. You want to be particular about which one this is about.